meteor/lib/rundown/rundownTiming.ts

```diff
@@ -181,12 +181,12 @@ export class RundownTimingCalculator {
 				if (partInstance.segmentId !== lastSegmentId) {
 					this.untimedSegments.add(partInstance.segmentId)
 					lastSegmentId = partInstance.segmentId
-					segmentDisplayDuration = 0
-					if (segmentBudgetDurationLeft > 0) {
-						waitAccumulator += segmentBudgetDurationLeft
-					}
-					segmentBudgetDurationLeft = this.segmentBudgetDurations[unprotectString(partInstance.segmentId)]
 				}
+				segmentDisplayDuration = 0
```

Comment:
I don't know if moving this assignment would have any implications as you now always sets it to zero.
But it seems like that "segmentDisplayDuration" is never assigned another value than zero, so the variable might be entirely redundant? (This is pure guesswork though)

Reply:
You are right, it's value is always always 0